### PR TITLE
fix scroll bar issue and enhance dummy app.

### PR DESCRIPTION
**Pull request by kuldipsinghrai**
#Fix#
@rox163 
@vesper2000 
@JasonYe007 
@dafortin

### CODE REVIEW

**Comment by kuldipsinghrai** [comment]
I can attach screenshots of the scroll bar but I am not sure how to add it here.

**Comment by rox163** [comment]
you can attach files in these comments

File: tests/dummy/app/pods/demo/template.hbs [hunk 1]
```diff
@@ -1,9 +1,39 @@
-  <h1>frost-sidebar testbed</h1>
-
+<div class="demo-instance" >
```
Comment (dafortin) [inline tests/dummy/app/pods/demo/template.hbs]: To me your indentation seems wrong.  Example Line 2 vs 3, line 5 vs 6, etc.

Comment (kuldipsinghrai) [inline tests/dummy/app/pods/demo/template.hbs]: fixed.

Comment (dafortin) [inline tests/dummy/app/pods/demo/template.hbs]: Did you double check all your files to make sure that the indentation is correct?

Comment (kuldipsinghrai) [inline tests/dummy/app/pods/demo/template.hbs]: just realized that and I am going through it.

Comment (dafortin) [inline tests/dummy/app/pods/demo/template.hbs]: great thanks !

**Pull request merged**